Makefile

```diff
@@ -33,3 +28,11 @@ install-piptools:
 doc-requirements.txt: doc-requirements.in install-piptools
 	$(call PIP_COMPILE,doc-requirements.in)
 
+.PHONY: requirements.txt
+requirements.txt: requirements.in install-piptools
+	$(call PIP_COMPILE,requirements.in)
+
+.PHONY: stats
+stats:
+	@generate-dashboard -o deployment/stats/prometheus/flytepropeller-dashboard.json stats/flytepropeller_dashboard.py
```

Comment:
where is generate-dashboard defined?

Reply:
its part of the requirements.txt. you have to install it